docs/spec/sanitations.md

```diff
@@ -28,7 +28,8 @@ These changes are done in order to improve the overall usability, and to address
     * users.deletePhoto
     * users.setPhoto
 
-2. Changed the schema name `Default success template` to `DefaultSuccessResponse` in the components section of the OpenAPI specification, to improve the usability of the counterpart Ballerina record name.
+2. Sanitized the inline and component schema names by removing the whitespaces, special characters and, converting them to pascal case.
+   * This was done using the `sanitations.bal` script under the `docs/spec` directory.
```

Comment:
Following up with this comment [1]; Here I think we can mention how we could execute the `sanitations.bal` using the gradle build.

[1] - https://github.com/ballerina-platform/module-ballerinax-slack/pull/164/files#r1674504405

Reply:
as per https://github.com/ballerina-platform/module-ballerinax-slack/pull/164#discussion_r1675072862, no need to mention it explicitly I guess.